goodput/src/worker.c

```diff
@@ -787,7 +801,7 @@ void dma_goodput(struct worker *info)
 	while (!info->stop_req) {
 		uint64_t cnt;
 
-		if (dma_tx_lat == info->action) {
+		if (dma_tx_lat == info->action && info->wr) {
```

Comment:
I was getting into here without info->ib_ptr.  I'm not sure this is correct, but it mirrors the check above at line ~764.